### DISCUSSION TITLE
Brew reinstall claims it backs up the app, but it just made an alias to the app?

### DISCUSSION THREAD

**Discussion by jasikpark** [Homebrew/discussions#1137] [Casks]
```sh
brew reinstall multimc
==> Downloading https://github.com/MultiMC/MultiMC5/releases/download/0.6.12/mmc
Already downloaded: /Users/calebjasik/Library/Caches/Homebrew/downloads/ee470de006ad4bf6435fd027ace90f66b64e3016dcd4b4c4e0e91a75053c8f19--mmc-stable-osx64.tar.gz
==> Uninstalling Cask multimc
==> Backing App 'MultiMC.app' up to '/usr/local/Caskroom/multimc/0.6.12/MultiMC.
==> Removing App '/Applications/MultiMC.app'
==> Purging files for version 0.6.12 of Cask multimc
==> Installing Cask multimc
==> Moving App 'MultiMC.app' to '/Applications/MultiMC.app'
🍺  multimc was successfully installed!
```

So I went to `/usr/local/Caskroom/multimc/0.6.12/MultiMC` and saw that the "backup" was an alias to `/Applications/MultiMC.app`

<img width="737" alt="image" src="https://user-images.githubusercontent.com/10626596/112684673-5a10c900-8e41-11eb-9427-13ddfca72f65.png">

Is this not meant to be a real backup?

**Comment by SMillerDev** [comment]
It's making a backup and purging that backup further in the flow. It's not meant as a backup for you to use.

**Comment by jasikpark** [comment]
Ah, ok. I never understood that from reading the output before, I’m doubly glad that I had a backup that I was restoring from then.

Caleb Jasik
***@***.***
https://jasik.xyz

On March 26, 2021 at 3:25:52 PM, Sean Molenaar ***@***.***) wrote:

It's making a backup and purging that backup further in the flow. It's not meant as a backup for you to use.
